docs/user_guide/mitigation.rst

```diff
@@ -74,6 +74,62 @@ Preprocessing
    
 .. currentmodule:: fairlearn.preprocessing
 
+Preprocessing algortihms transform the dataset to mitigate possible unfairness
+present in the data.
+Preprocessing algorithms in Fairlearn follow the `scikit-learn transformers
+<https://scikit-learn.org/stable/modules/generated/sklearn.base.TransformerMixin.html#sklearn.base.TransformerMixin>`_
+class, meaning that they can :code:`fit` to the dataset and :code:`transform` it.
+A :code:`fit_transform` might not always be available, in which case the separate
+:code:`fit` and :code:`transform` methods need to be used in sequence.
+
+.. _correlation_remover:
+
+Correlation Remover
+~~~~~~~~~~~~~~~~~~~
+Sensitive features can be correlated with non sensitive features in the dataset.
+By applying the :code:`CorrelationRemover`, these correlations are filtered out while
+as much as possible information is retained (as measured by the least-squares error).
```

Comment:
I think we should either mention the `alpha` parameter here or include something about how you can control the level of "filtering" of correlation, so users understand it's not always complete removal of correlation.

Reply:
I agree with both these comments! I've mentioned the `alpha` parameter earler in the doc now, as well as using a different wording for "information".